src/analyses/creationLocksetAlternative.ml

```diff
@@ -0,0 +1,210 @@
+(** alternative creation lockset analysis
```

Comment:
At least looking at LOCs, this one does not seem much more complicated than the first option.